### PR TITLE
Add User-Agent header to API requests - Closes #651

### LINKED ISSUE #651: Change os header in api
**Expected behavior:** We should use HTTP headers properly.

**Actual behavior:** We send `lisk-js-api` as an `os` header so the server can spot our requests. This isn't very semantic.

### PR DESCRIPTION
**Pull request by willclarktech**
### What was the problem?

We set an `os` header for some reason.

### How did I fix it?

Set `User-Agent` instead.

### How to test it?

Send requests to a node and check the headers

### Review checklist

* The PR solves #651 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the
	[commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated

### CODE REVIEW

File: src/api_client/api_client.js [hunk 1]
```diff
@@ -38,7 +38,7 @@ const defaultOptions = {
 
 const commonHeaders = {
 	'Content-Type': 'application/json',
-	os: 'lisk-js-api',
+	'User-Agent': 'lisk-js/1.0',
```
Comment (nazarhussain) [inline src/api_client/api_client.js]: @willclarktech This change is good but should't we change it to **LiskElements**? Or that will be done in one transition? 

Also one change I could suggest here. The standard `User-Agent` sting as following pattern: 

```
[ProductName]/[version] ([system and browser information]) [platform] ([platform details]) [extensions].
```

So here I believe; 

```
ProductName: LiskElements
Version: 1.0
System Information: If we can add we should 
Platform: We can add this information easily 
```

One last thing around all `UserAgents` I see every one uses `CamelCase` for Product Name, should not we follow it as well? e.g. Mozilla, AppleWebKit, Googlebot

If you think its not a browser, even then custom agents follow the same pattern.e.g. 

```
Googlebot/2.1 (+http://www.google.com/bot.html)
```

Comment (slaweet) [reply src/api_client/api_client.js]: I would consider getting the version number from `package.json` (crop to only major and minor version) to keep it up-to-date.

Comment (shuse2) [reply src/api_client/api_client.js]: I don't think we should add `lisk-js` version number.
Lisk-js APIClient is just a library and not actual client imo

Comment (nazarhussain) [reply src/api_client/api_client.js]: @shuse2 That APIClient is based on an engine/library/module `lisk-js`which have a specific version number. And for debugging and analytics its a valuable information.

Comment (willclarktech) [reply src/api_client/api_client.js]: Let's arrange a meeting to discuss this.

File: src/api_client/api_client.js [hunk 2]
```diff
@@ -37,8 +38,25 @@ const defaultOptions = {
...
+		process.env.LC_ALL ||
+		process.env.LC_MESSAGES ||
+		process.env.LANG ||
+		process.env.LANGUAGE;
+	const systemInformation = `${os.platform()} ${os.release()}; ${os.arch()}${
+		locale ? `; ${locale}` : ''
+	}`;
+	return `${name}/${version} (${engine}) ${liskJSInformation} ${
+		systemInformation
+	}`;
```
Comment (nazarhussain) [inline src/api_client/api_client.js]: There should be a `,` between lisk and system information.

Comment (willclarktech) [reply src/api_client/api_client.js]: Do you have any examples of this as a standard? It doesn't appear anywhere here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

**Pull request merged**